lib/osbuild.py

```diff
@@ -350,7 +351,20 @@ def create(self, *, blueprint=None, bp_verbatim=None, profile=None):
 
             bp_name = stack.enter_context(blueprint.to_composer())
 
-            composer_cli('blueprints', 'depsolve', bp_name)
+            # re-try multiple times to try to avoid a bug:
+            # ERROR: Depsolve Error: Get "http://localhost/.../depsolve/contest_blueprint": EOF
```

Comment:
Is the bug reported somewhere? Or should we report it to osbuild?

Reply:
I'm pretty sure it is, under some appropriate `osbuild*` component (not sure which one now), but there are many similar (arguably higher-severity) bugs rotting there, the team doesn't really seem to be managing the bug queue, hence the workaround.